dbt/adapters/postgres.py

```diff
@@ -291,7 +291,7 @@ def rename(cls, profile, from_name, to_name, model_name=None):
 
     @classmethod
     def execute_model(cls, profile, model):
-        parts = re.split(r'-- (DBT_OPERATION .*)', model.compiled_contents)
+        parts = re.split(r'-- (DBT_OPERATION .*)', model.get('wrapped_sql'))
```

Comment:
we'll be able to replace this with separate nodes grouped into a transaction, right?